--- ./bin/cfgmaker.org	2006-02-03 07:33:33.000000000 +0100
+++ ./bin/cfgmaker	2006-03-27 11:46:16.164956480 +0200
@@ -469,7 +469,7 @@
                 my $if_is_loopback = $if_type == 24;
                 my $if_is_ciscovlan =
                     ($$router_dev{Vendor} eq 'cisco'
-                    and $$i{Descr} =~ /^V(LAN|lan)\d+$/);
+                    and $$i{Descr} =~ /^(unrouted )?V(LAN|lan)\ ?\d+$/);
 
                 my $if_ip = $$i{Ip};
                 my $if_snmp_descr = $$i{Descr};
