diff -ur mrtg-2.9.18.org/Makefile.in mrtg-2.9.18/Makefile.in
--- mrtg-2.9.18.org/Makefile.in	Fri May 31 13:39:12 2002
+++ mrtg-2.9.18/Makefile.in	Sun Jun  2 20:43:07 2002
@@ -45,7 +45,7 @@
 	  $(INSTALL) -m 755 $$x $(prefix)/bin; done
 	for x in bin/rateup; do \
 	  $(INSTALL) -m 755 $$x $(prefix)/bin; done
-	$(MKINSTALLDIRS) $(prefix)/lib/mrtg2/Pod
+	$(MKINSTALLDIRS) $(prefix)/lib/mrtg2
 	for x in $(top_srcdir)/lib/mrtg2/*.pm; do \
 	  $(INSTALL) -m 644 $$x $(prefix)/lib/mrtg2; done
 	for x in $(top_srcdir)/lib/mrtg2/Pod/*.pm; do \
Only in mrtg-2.9.18: Makefile.in~
