diff -urN mrtg-2.10.14.orig/lib/mrtg2/locales_mrtg.pm mrtg-2.10.14/lib/mrtg2/locales_mrtg.pm
--- mrtg-2.10.14.orig/lib/mrtg2/locales_mrtg.pm	2004-06-08 23:08:31.000000000 +0200
+++ mrtg-2.10.14/lib/mrtg2/locales_mrtg.pm	2004-07-28 18:52:38.000000000 +0200
@@ -3819,7 +3819,7 @@
       'Jan'       => 'Sty',          'Feb'       => 'Lut',         'Mar'       => 'Mar',
       'April'     => 'Kwietnia',     'May'       => 'Maja',        'June'      => 'Czerwca', 
       'Apr'       => 'Kwi',          'May'       => 'Maj',         'Jun'       => 'Cze',
-      'July'      => 'Lipica',       'August'    => 'Sierpnia',    'September' => 'Wrzesnia', 
+      'July'      => 'Lipca',        'August'    => 'Sierpnia',    'September' => 'Wrze¶nia', 
       'Jul'       => 'Lip',          'Aug'       => 'Sie',         'Sep'       => 'Wrz', 
       'October'   => 'Pa¼dziernika', 'November'  => 'Listopada',   'December'  => 'Grudnia', 
       'Oct'       => 'Pa¼',          'Nov'       => 'Lis',         'Dec'       => 'Gru' 
diff -urN mrtg-2.10.14.orig/translate/locales_mrtg.pm mrtg-2.10.14/translate/locales_mrtg.pm
--- mrtg-2.10.14.orig/translate/locales_mrtg.pm	2004-06-08 23:08:31.000000000 +0200
+++ mrtg-2.10.14/translate/locales_mrtg.pm	2004-07-28 18:52:32.000000000 +0200
@@ -3819,7 +3819,7 @@
       'Jan'       => 'Sty',          'Feb'       => 'Lut',         'Mar'       => 'Mar',
       'April'     => 'Kwietnia',     'May'       => 'Maja',        'June'      => 'Czerwca', 
       'Apr'       => 'Kwi',          'May'       => 'Maj',         'Jun'       => 'Cze',
-      'July'      => 'Lipica',       'August'    => 'Sierpnia',    'September' => 'Wrzesnia', 
+      'July'      => 'Lipca',        'August'    => 'Sierpnia',    'September' => 'Wrze¶nia', 
       'Jul'       => 'Lip',          'Aug'       => 'Sie',         'Sep'       => 'Wrz', 
       'October'   => 'Pa¼dziernika', 'November'  => 'Listopada',   'December'  => 'Grudnia', 
       'Oct'       => 'Pa¼',          'Nov'       => 'Lis',         'Dec'       => 'Gru' 
diff -urN mrtg-2.10.14.orig/translate/polish.pmd mrtg-2.10.14/translate/polish.pmd
--- mrtg-2.10.14.orig/translate/polish.pmd	2003-10-26 13:11:56.000000000 +0100
+++ mrtg-2.10.14/translate/polish.pmd	2004-07-28 18:52:15.000000000 +0200
@@ -96,7 +96,7 @@
       'Jan'       => 'Sty',          'Feb'       => 'Lut',         'Mar'       => 'Mar',
       'April'     => 'Kwietnia',     'May'       => 'Maja',        'June'      => 'Czerwca', 
       'Apr'       => 'Kwi',          'May'       => 'Maj',         'Jun'       => 'Cze',
-      'July'      => 'Lipica',       'August'    => 'Sierpnia',    'September' => 'Wrzesnia', 
+      'July'      => 'Lipca',        'August'    => 'Sierpnia',    'September' => 'Wrze¶nia', 
       'Jul'       => 'Lip',          'Aug'       => 'Sie',         'Sep'       => 'Wrz', 
       'October'   => 'Pa¼dziernika', 'November'  => 'Listopada',   'December'  => 'Grudnia', 
       'Oct'       => 'Pa¼',          'Nov'       => 'Lis',         'Dec'       => 'Gru' 
