diff -urN mrtg-2.6.4.orig/contrib/cpumon/cpumon.pl mrtg-2.6.4/contrib/cpumon/cpumon.pl
--- mrtg-2.6.4.orig/contrib/cpumon/cpumon.pl	Sat Feb 20 21:43:27 1999
+++ mrtg-2.6.4/contrib/cpumon/cpumon.pl	Sun Feb 21 14:16:14 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # This was taken from a PERL script Chris Behrens wrote to monitor
 # resource usage for his IRC servers and was trimmed down to
diff -urN mrtg-2.6.4.orig/contrib/distrib/distrib.pl mrtg-2.6.4/contrib/distrib/distrib.pl
--- mrtg-2.6.4.orig/contrib/distrib/distrib.pl	Sat Feb 20 21:43:28 1999
+++ mrtg-2.6.4/contrib/distrib/distrib.pl	Sun Feb 21 14:16:33 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 # version 1.1, 27.06.97, philippe.simonet@swisstelecom.com
 
 $outFile = 'distrib.html';
diff -urN mrtg-2.6.4.orig/contrib/get-active/get-active.pl mrtg-2.6.4/contrib/get-active/get-active.pl
--- mrtg-2.6.4.orig/contrib/get-active/get-active.pl	Sat Feb 20 21:43:28 1999
+++ mrtg-2.6.4/contrib/get-active/get-active.pl	Sun Feb 21 14:16:47 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # Author: Russ Wright, Lawrence Berkeley National Laboratory December 1997
 #
diff -urN mrtg-2.6.4.orig/contrib/get-multiserial/get-modems.pl mrtg-2.6.4/contrib/get-multiserial/get-modems.pl
--- mrtg-2.6.4.orig/contrib/get-multiserial/get-modems.pl	Sat Feb 20 21:43:28 1999
+++ mrtg-2.6.4/contrib/get-multiserial/get-modems.pl	Sun Feb 21 14:17:04 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # Returns number of active analog and digital dial-ins
 #
diff -urN mrtg-2.6.4.orig/contrib/ircstats/ircstats.pl mrtg-2.6.4/contrib/ircstats/ircstats.pl
--- mrtg-2.6.4.orig/contrib/ircstats/ircstats.pl	Sat Feb 20 21:43:28 1999
+++ mrtg-2.6.4/contrib/ircstats/ircstats.pl	Sun Feb 21 14:17:17 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl5
+#!/usr/bin/perl
 # client.pl  v1.5 7 Oct 1997 Matthew Ramsey <mjr@blackened.com>
 # http://www.blackened.com/ircstats/
 # A simple perl script I use to grab client info on IRC Servers.  While
diff -urN mrtg-2.6.4.orig/contrib/mrtg-dynip/mrtg-dynip mrtg-2.6.4/contrib/mrtg-dynip/mrtg-dynip
--- mrtg-2.6.4.orig/contrib/mrtg-dynip/mrtg-dynip	Sat Feb 20 21:43:28 1999
+++ mrtg-2.6.4/contrib/mrtg-dynip/mrtg-dynip	Sun Feb 21 14:17:29 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 ##################################################################
 # Multi Dynamic IP Traffic Grapher contibution to mrtg
 ##################################################################
diff -urN mrtg-2.6.4.orig/contrib/mrtg-dynip/mrtg-dynip-sydnet mrtg-2.6.4/contrib/mrtg-dynip/mrtg-dynip-sydnet
--- mrtg-2.6.4.orig/contrib/mrtg-dynip/mrtg-dynip-sydnet	Sat Feb 20 21:43:28 1999
+++ mrtg-2.6.4/contrib/mrtg-dynip/mrtg-dynip-sydnet	Sun Feb 21 14:17:38 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 ##################################################################
 # Multi Dynamic IP Traffic Grapher contibution to mrtg
 ##################################################################
diff -urN mrtg-2.6.4.orig/contrib/mrtg-mail/mailstats mrtg-2.6.4/contrib/mrtg-mail/mailstats
--- mrtg-2.6.4.orig/contrib/mrtg-mail/mailstats	Sat Feb 20 21:43:28 1999
+++ mrtg-2.6.4/contrib/mrtg-mail/mailstats	Sun Feb 21 14:17:53 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 #
 # Author:  Petter Reinholdtsen <pere@td.org.uit.no>
 # Date:    1997-07-09
diff -urN mrtg-2.6.4.orig/contrib/mrtgindex.cgi/mrtgindx.cgi mrtg-2.6.4/contrib/mrtgindex.cgi/mrtgindx.cgi
--- mrtg-2.6.4.orig/contrib/mrtgindex.cgi/mrtgindx.cgi	Sat Feb 20 21:43:28 1999
+++ mrtg-2.6.4/contrib/mrtgindex.cgi/mrtgindx.cgi	Sun Feb 21 14:18:05 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # mrtgindex.cgi v 1.1
 # (c) 1997, Mick Ghazey mick@lowdown.com
